source/sass/type.scss

```diff
@@ -70,7 +72,7 @@ h1,
   font-weight: 400;
   letter-spacing: 0;
 
-  @include scaleText(36px, 40px, 56px, 64px);
+  @include scaleText(36px, 40px, $h1-font-size, 64px);
```

Comment:
Why only the  third scale? Should the other values be based on `$h1-font-size`, too? (like x2/3,  x3/4, x1, x1.15?)

Reply:
Turned 56px which is the font size we set for desktop into a variable so I can use that variable to position the `.copy-link` properly in `product.scss`

Reply:
If we don't want to touch this `type.scss` I can just hardcode 

```
.copy-link {
  position: absolute;
  top: 28px; /* instead of $h1-font-size/2 */
  right: 0;
}
```

Reply:
It feels a bit bug-introducing if we only change one of the four values from number to variable, so if we can turn the others into variables as well, that'd be nicer.